### ISSUE TITLE
Featured Story Promo Banner

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1399]
**Block:** Columns (highlight) (replaces carousel 'featured stories' on current home page docx + columns style)

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=1112-2418&t=EsHkGf7BvtaFQtnM-0

**User Story:** As a site visitor, I want to see featured stories that are important to the community so that I can learn more about topics I'm invested in.

**Requirements:** 
1. Image aligns left
2. Add styles for secondary blue (#243772) and primary orange (#963821) color variations
3. All text is white with Eyebrow using eyebrow styling, title H2, and description text as Body_M
4. Regardless of alignment style, image stacks on top of text on mobile
5. Author should be able to optionally add a button

- Button displays with white background and primary (blue or orange depending on banner style applied) text color
- Displays at bottom of text area, right aligned

6. Any hyperlink added (to asset or in description text) should be keyboard navigable
7. Screen reader should announce the banner content in logical page order

**Acceptance Criteria:** 
1. Given I am authoring a featured story, when I change the styles, then I am able to change the background color for the text area to blue or primary orange. 
2. Given I am viewing a featured story, when the content renders, the the text displays all white and eyebrow leverages eyebrow style, title leverages H2 styling, and description leverages body_M.
3. Given I am viewing a featured story, when the content renders, then the image stacks on top regardless of alignment style. 
4. Given I am authoring a featured story, when I edit the component block, then I am able to add a hyperlink to the image if needed. 
5. Given I am using my keyboard to navigate, when there is a button present in the featured story, then I am able to keyboard navigate to the button within the featured story. 
6. Given I am using a screen reader, when I encounter a featured story, then the content is announced in logical page order with the correct ARIA tags. 

**QA Testing:** 
Authoring link: https://clarkcountynv.sharepoint.com/:w:/r/teams/AEM/_layouts/15/Doc.aspx?sourcedoc=%7B82215B7F-F516-441D-B190-08F4C59D24A2%7D&file=columns-featured-story.docx&action=default&mobileredirect=true

**Comment by sarah0arnold** [comment]
Approved in UAT demo on 8/4/25 with no feedback. Closing story
